src/main/java/frc/robot/Constants.java

```diff
@@ -18,22 +20,27 @@
     // Actual IDs on robot, used to activate the right motors
     // TODO: Values to be changed, these are placeholder values for now
     public static final int kDriveRightFront = 0;
-    public static final int kDriveRightMiddle = 0;
     public static final int kDriveRightRear = 0;
     public static final int kDriveLeftFront = 0;
-    public static final int kDriveLeftMiddle = 0;
     public static final int kDriveLeftRear = 0;
```

Comment:
It was a good thought to zero out the id's, but it'll cause code to crash if we try to use this to test something else (or run simulation). Set these to unique numbers.

Reply:
These were updated over the weekend, should be in Main if you'll merge that in